spec/controllers/works/default_rails_actions_spec.rb

```diff
@@ -350,6 +351,14 @@ def call_with_params(params)
       expect(assigns(:fandom)).to eq(@fandom)
     end
 
+    describe "when the fandom id is invalid" do
+      it "causes a 404 error" do
+        params = { fandom_id: @invalid_fandom_id }
```

Comment:
Please use `0` for the id here, it's a bit simpler.

Reply:
Done in 0854987af09831c12415170578c1e1f050c1a4ef